src/main/java/cn/edu/thu/tsfile/common/conf/TSFileDescriptor.java

```diff
@@ -60,22 +60,66 @@ private void loadYaml() {
 				System.exit(1);
 			}
 		}
-		LOGGER.info("start to read config file {}", url);
+		LOGGER.info("Start to read config file {}", url);
+		Properties properties = new Properties();
 		try {
-			Yaml yaml = new Yaml();
-			conf = yaml.loadAs(inputStream, TSFileConfig.class);
-		} catch (Exception e) {
-			LOGGER.error("Loading settings {} failed.", url, e);
-			System.exit(1);
-		} finally {
-			if (inputStream != null) {
-				try {
-					inputStream.close();
-					inputStream = null;
-				} catch (IOException e) {
-				}
-			}
+		    properties.load(inputStream);
+		    
+		    String rowGroupSize = properties.getProperty("rowGroupSize");

```

Comment:
Besides, I agree with @liukun4515 , default value is important I think. Otherwise we need provide a conf/tsfile.properties.template or something.